M2/Macaulay2/e/NCAlgebras/FreeAlgebraQuotient.hpp

```diff
@@ -78,7 +78,13 @@ class FreeAlgebraQuotient : public our_new_delete
                      bool p_parens) const;
 
   bool is_homogeneous(const Poly& f) const;
-  void degree(const Poly& f, monomial d) const;
+  // FIXME: copied from ring.hpp because this doesn't inherit from Ring
```

Comment:
@mikestillman @moorewf Here (and in `FreeAlgebra.hpp`) I had to duplicate an inline function from `ring.hpp` because `FreeAlgebra` and `FreeAlgebraQuotient` don't inherit from each other or from `Ring`. Is there a reason for this?